sql/core/src/test/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister

```diff
@@ -2,6 +2,7 @@ org.apache.spark.sql.sources.FakeSourceOne
 org.apache.spark.sql.sources.FakeSourceTwo
 org.apache.spark.sql.sources.FakeSourceThree
 org.apache.spark.sql.sources.FakeSourceFour
+org.apache.spark.sql.sources.CommandRunnableDataSource
```

Comment:
we don't need to register it as we are not testing the short name. We can just use the full class name when calling `spark.executeCommand`